modules/repository-base/base/Makefile

```diff
@@ -58,6 +58,7 @@ endif
 ##################################
 
 VERSION ?= $(shell git describe --tags --always --match='v*' --abbrev=14 --dirty)
+IS_PRERELEASE := $(shell echo "$(VERSION)" | grep -q '-' && echo true || echo false)
```

Comment:
suggestion: Is this safe? Imagining we're releasing a new version `v50.0.0`; if part of one of the modules started to (accidentally) make the git state dirty during a release, this would make the version show as a prerelease.

E.g.

```console
$ git tag -s -m"v50" v50.0.0
$ echo test > hello && git add hello
$ git describe --tags --always --match='v*' --abbrev=14 --dirty
v50.0.0-dirty
```

Because of the dash, this would be marked as a prerelease version. It's easy to imagine how this could happen.

There are a lot of ways to solve this - we probably should cancel a release if the git tag is dirty, we could change this parsing to check for `-alpha` or `-beta`, we could not use `--dirty` in VERSION or we could have `RELEASE_VERSION` which doesn't use `--dirty`.

What do you think? Is this a concern?